src/styles/blocks/header/variables.scss

```diff
@@ -0,0 +1,2 @@
+$header_height: 60px;
+$blue_color: #00acdc;

```

Comment:
Keep the variable in one place, also $blue_color is not an informative name, the variable should show its purpose.

For example, you have in other folder variables too, and $button-color-main: #00acdc; is the same, so if you need to change the color you have to do it in two places.